src/xpath.h

```diff
@@ -80,6 +80,9 @@
 #define LYXP_STRING_CAST_SIZE_START 64
 #define LYXP_STRING_CAST_SIZE_STEP 16
 
+/* Number of node checks when it starts being worth using the string dict */
```

Comment:
Like I said, please be a bit more verbose. Something about node name matching with strcmp on few nodes being faster than inserting it into the dictionary and then comparing pointers.

Reply:
Ok.